portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+body {
+  background-image: url('images/white-cat-1.png');
+}
+</style>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>The Portfolio</h1>
+      <b>(Scroll to the bottom for a surprise)</b>
+      <p>Hello, I am Anthony Achille, Check out my <a href="https://github.com/Anthony869?tab=repositories">GitHub</a></p>
```

Comment:
This would also be an opportunity to add your Instagram/Snapchat "handle" (user) if you wanted to.  It is up to you.

I encourage you, as you work through this, to really set this up a personal/professional profile, that you could point people to.  (We can worry about the URL later).  It can be a pretty powerful site to kind of show off your skills, and what you enjoy doing.  Similar to like your computer science teachers ipad site.